### PR TITLE
Please add me to the project 

### PR DESCRIPTION
**Pull request by Shaman-1975**
thank you

### CODE REVIEW

**Review by ProsperousHeart — APPROVED** [review]
Approved

**Review by ProsperousHeart — APPROVED** [review]
Approved!

File: Submissions/Shaman-1975.json [hunk 1]
```diff
@@ -0,0 +1,16 @@
...
+    "img": "https://avatars.githubusercontent.com/u/58999272?v=4",
+    "email": "claude@chi.lu",
+    "links": {
+      "website": "webthings.lu",
+      "linkedin": "https://www.linkedin.com/in/claude-dumont",
+      "github": "https://github.com/Shaman-1975"
+    },
+    "jobTitle": "Senior FrontEnd Developper",
+    "location": {
+      "city": "Kavala",
```
Comment (Shaman-1975) [inline Submissions/Shaman-1975.json:12]: 13 + "city": "Aiandeo" ,

**Review by Shaman-1975 — COMMENTED** [review]
I moved again @claude@chi.lu

**Pull request merged**